doc/sphinx/source/conf.py

```diff
@@ -420,7 +420,7 @@
 
 # Configuration for intersphinx
 intersphinx_mapping = {
-    'cf_units': ('https://cf-units.readthedocs.io/en/latest/', None),
+    'cf_units': ('https://scitools.org.uk/cf-units/docs/latest/', None),
```

Comment:
@stacristo Why did you change this url? As I understand is the error message you got because of this new url. The older one should work.